apps/etl-func/src/adapters/eventhub/config.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+export const eventhubConfigSchema = z.object({
+  connectionUri: z.string(),
```

Comment:
Maybe we can use something like `.url()` or at leas add `.min(1)`

Reply:
we can use a regex to check for `sb://<NamespaceName>.servicebus.windows.net`